tcms/telemetry/static/telemetry/js/testing/breakdown.js

```diff
@@ -1,39 +1,18 @@
 $(document).ready(() => {
     $('.selectpicker').selectpicker();
 
-    loadInitialProduct();
+    loadInitialProduct(reloadCharts);
 
     $('#id_after').on('dp.change', reloadCharts);
     $('#id_before').on('dp.change', reloadCharts);
-    document.getElementById('id_select_test_plan').onchange = reloadCharts;
-    document.getElementById('id_select_product').onchange = updateTestPlanSelect;
+    document.getElementById('id_test_plan').onchange = reloadCharts;
+    document.getElementById('id_select_product').onchange = () => updateTestPlanSelectFromProduct(reloadCharts);
```

Comment:
Not sure what is going on here but:
- initially I don't see any test plans, not a big deal but maybe a sign of a problem
- when Product is selected neither of Test plan, Version nor Build is updated.

Reply:
> initially I don't see any test plans, not a big deal but maybe a sign of a problem

There was no such functionality, now there is

> when Product is selected neither of Test plan, Version nor Build is updated

this is related to the 1000th times I renamed the selectors and had a selector mismatch. All is fine now